src/sagemaker/workflow/function_step.py

```diff
@@ -100,6 +101,9 @@ def __init__(
         self._step_kwargs = kwargs
 
         self.__job_settings = None
+        self._properties = Properties(
+            step_name=name, step=self, shape_name="DescribeTrainingJobResponse"
+        )
```

Comment:
It's for internal usage to retrieve execution id from the properties.
However, following the current practice, we won't expose the properties of function step to customers